charts/homepage/templates/rbac.yaml

```diff
@@ -25,27 +25,28 @@ rules:
       - get
       - list
   - apiGroups:
-      - traefik.containo.us
       - traefik.io
     resources:
       - ingressroutes
     verbs:
       - get
       - list
   - apiGroups:
-      - metrics.k8s.io
+      - gateway.networking.k8s.io
     resources:
-      - nodes
-      - pods
+      - httproutes
+      - gateways
     verbs:
       - get
       - list
   - apiGroups:
-      - apiextensions.k8s.io
```

Comment:
This is still used
https://github.com/gethomepage/homepage/blob/dev/src/utils/config/kubernetes.js#L37